backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -82,11 +89,15 @@ public TestEvent(TestOrder order, Boolean hasPriorTests) {
       // this can happen during unit tests, but never in prod.
       log.error("Order {} missing PatientAnswers", order.getInternalId());
     }
+    this.results = new HashSet<>();
```

Comment:
to confirm... this was to prevent the scenarios where we were initializing `TestEvent`s that had `null` `results`?